testdata/cloudflare/cloudflare_firewall_rule.yaml

```diff
@@ -34,7 +34,14 @@ interactions:
               "ref": "FIL-100"
             }
           }
-        ]
+        ],
+        "result_info": {
+          "page": 1,
+          "per_page": 100,
+          "count": 0,
+          "total_count": 0,
+          "total_pages": 0
```

Comment:
this is actually an invalid payload and running the pagination will fail. the `count`, `total_count` and `total_pages` are used in the paginator to move to the next set of results.

Reply:
Done and done.